contracts/mock/KyberNetworkTokenV3.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+
+import '@openzeppelin/contracts/access/Ownable.sol';
+import '@openzeppelin/contracts/token/ERC20/ERC20Burnable.sol';
+import '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import '@openzeppelin/contracts/token/ERC20/SafeERC20.sol';
+
+interface IERC20Burnable {
+  function burnFrom(address _from, uint256 _value) external;
+}
+
+/// @dev copy from kyber network repo
+contract KyberNetworkTokenV3 is ERC20Burnable, Ownable {
```

Comment:
maybe rename to MockTokenSmallDecimals... more descriptive. Or USDC :p

Reply:
MockTokenSmallDecimals sounds good. tks